src/field/arena.ts

```diff
@@ -10,7 +10,7 @@ import Move from "#app/data/move";
 import { ArenaTag, ArenaTagSide, ArenaTrapTag, getArenaTag } from "#app/data/arena-tag";
 import { BattlerIndex } from "#app/battle";
 import { Terrain, TerrainType } from "#app/data/terrain";
-import { applyPostTerrainChangeAbAttrs, applyPostWeatherChangeAbAttrs, PostTerrainChangeAbAttr, PostWeatherChangeAbAttr } from "#app/data/ability";
+import { applyAbAttrs, applyPostTerrainChangeAbAttrs, applyPostWeatherChangeAbAttrs, PostTerrainChangeAbAttr, PostWeatherChangeAbAttr, TerrainEventTypeChangeAbAttr } from "#app/data/ability";
```

Comment:
```suggestion
import {
  applyAbAttrs,
  applyPostTerrainChangeAbAttrs,
  applyPostWeatherChangeAbAttrs,
  PostTerrainChangeAbAttr,
  PostWeatherChangeAbAttr,
  TerrainEventTypeChangeAbAttr
} from "#app/data/ability";
```
Import lines are getting way too long 😓